loopout.go

```diff
@@ -1533,6 +1536,32 @@ func (s *loopOutSwap) clampSweepFee(fee btcutil.Amount) (btcutil.Amount, bool) {
 func (s *loopOutSwap) sweepMuSig2(ctx context.Context,
 	htlcOutpoint wire.OutPoint, htlcValue btcutil.Amount) bool {
 
+	// If we're using a wallet address as a destination address then this
+	// sweep can be batched with other sweeps.
+	if s.IsWalletAddr {
```

Comment:
Q: Is the sweep batcher enabled by default? Are there scenarios where users don't want to batch sweeps meaning, should we provide a flag to enable it?

Reply:
Yes it is enabled by default, I agree we could expose a flag for enabling/disabling the batcher